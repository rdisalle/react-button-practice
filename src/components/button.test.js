import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Button from './button';

describe('<Button />', () => {
    it('renders empty', () => {
        const wrapper = shallow(<Button />)
        expect(toJson(wrapper)).toMatchSnapshot()
      })

    it('Should call props.onClick when the button is clicked', () => {
        const wrapper = shallow(<Button />);
        wrapper.find('button').at(0).simulate('click');
        expect(toJson(wrapper)).toMatchSnapshot()
    });
});
