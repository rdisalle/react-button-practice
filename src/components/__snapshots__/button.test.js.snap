// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders empty 1`] = `
<button
  onClick={[Function]}
>
  Click me!
</button>
`;
